vmx/src/vm.rs

```diff
@@ -4,9 +4,15 @@
 //
 // Copyright (c) DUSK NETWORK. All rights reserved.
 
+mod native;
+pub use native::NativeQuery;
+
 use std::collections::BTreeMap;
+use std::path::Path;
 
 use bytecheck::CheckBytes;
+use dallo::ModuleId;
+use native::NativeQueries;
```

Comment:
This is a leftover from the "hatchery" where a `host_host_query` function naturally arose, and we were "forced" to chose different nomenclature. We should probably go back to "host" here, since native is absolutely synonymous.